micro_sam/training/sam_trainer.py

```diff
@@ -242,11 +244,17 @@ def _update_masks(self, batched_inputs, y, sampled_binary_y, sampled_ids, num_su
         return loss, mask_loss, iou_regression_loss, mean_model_iou
 
     def _get_updated_points_per_mask_per_subiter(self, masks, sampled_binary_y, batched_inputs, logits_masks):
+        # function to adjust the samples based on the longest side resizing
+        transform = ResizeLongestSide(self.model.sam.image_encoder.img_size)
```

Comment:
You don't need this. You can just use `self.model.transform`. That guarantees that we use the same transform everywhere.

Reply:
Makes sense. Updated it in the latest commit. Thanks!